src/tiler/sql/user_reservations.sql

```diff
@@ -0,0 +1,9 @@
+(SELECT
```

Comment:
for my education, how does this file get used? I can't find a reference to `user_reservations` in this diff.